packages/yew/src/lib.rs

```diff
@@ -26,7 +26,6 @@
 //! - `csr`: Enables Client-side Rendering support and [`Renderer`]. Only enable this feature if you
 //!   are making a Yew application (not a library).
 //! - `ssr`: Enables Server-side Rendering support and [`ServerRenderer`].
-//! - `tokio`: Enables future-based APIs on non-wasm32 targets with tokio runtime.
```

Comment:
Both `tokio` on `wasi` with threading support and `async-std`'s `spawn_local` are unstable.
`tokio` and `wasm-bindgen-futures` are going to be the only runtime that is supported by `prokio` for the moment.

Hence, I feel it might be better to forego the `tokio` feature until we need it.

Reply:
I like this this way. Tokio is hidden behind the pseudo-runtime (prokio) and doesn't need to be exposed here.